core/services/fluxmonitor/flux_monitor.go

```diff
@@ -639,9 +656,13 @@ var (
 func (p *PollingDeviationChecker) checkEligibilityAndAggregatorFunding(roundState contracts.FluxAggregatorRoundState) error {
 	if !roundState.EligibleToSubmit {
 		return ErrNotEligible
-	} else if roundState.AvailableFunds.Cmp(roundState.PaymentAmount) < 0 {
+	}
+	if roundState.AvailableFunds.Cmp(roundState.PaymentAmount) < 0 {
```

Comment:
Why the change?

Reply:
I needed it for debugging. Gratuitous `else if` chains should be cleansed with fire, because they make the code inflexible. I'll remove this, though.